src/modules/auth/ui/views/sign-up-views.tsx

```
@@ -0,0 +1,233 @@
+
+const formSchema = z.object({
+    name: z.string().min(1,{message: "Name is required"}),
+    email: z.string().email(),
+    password: z.string().min(1, { message: "Password is required" }),            
+    confirmPassword: z.string().min(1, { message: "Password is required" }),            //we are asking for new password
+})
+.refine((data)=>data.password===data.confirmPassword,{   
+    message: "Password don't match",
+    path:["confirmPassword"],
+});
+{/*Checks if the passord and confirm password is coorect */}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.

(lint/complexity/noUselessLoneBlockStatements)